### DISCUSSION TITLE
24/11/08 온라인 멘토링

### DISCUSSION THREAD

**Discussion by gimsoomin** [cloud2scape/.github#33] [회의록]
## 다음 일정

24/11/11 오후 3시경
- [ ] 강사님 알림
- [ ] 5시에 일일보고 이므로 그전에 보고서 작성 필요

- [x] 깃허브 및 노션 초대
- [ ] 슬랙 미초대 사유 설명

## 멘토링시 논의할 것들
- [ ] 포팅 상황 공유

### 멘토
- [ ] 나연 - 클라우드 포메이션 yaml 템플릿 완성중 
- [ ] 수민 - db replication 구성 불가로 인해 테라폼 hcl 학습중
- [ ] 찬준 - 
- [ ] 재욱 - 인증 구현 완료, 현재 eks 구성이 되지 않아 이미지를 만들었지만 올리지 못하는 상황

* aws opensearch를 한달동안 켜두면 20만원이상 나오므로 최종 데모에만 붙일 것
* wbs 가 지켜지지 않고 있지만 일단 다음주에 수동이든 자동이든 eks에 서비스가 포팅돼서 올라가면 문제는 없을 듯
* 포팅 후 부하테스트나 롤링 패치 등을 논의할 수 있을 
* 스프링 게이트웨이 포팅 문제는 일단 이슈를 파악하고 있다는 점에서 긍정적. eks에 올릴때 구성이슈가 더 생길 것
* 클라우드 비용에 대한 감과 절감이 필요함 - rds를 최소노드로 올릴 시 5~6만정도 나올것, 잘 아껴서 쓰면 평소에 비용때문에 쓰지 못한 서비스를 써볼 수 있을

### 과제
- [ ] aws 계정나오면 eks에 api 워드프레스 gw lb container 2개 rds 1개를 띄우는 튜토리얼 
- [ ] 수민 - hcl 로 cd 구현, rds 최소사양 확인
- [ ] 나연 - 11/08 데드라인으로 eks를 구성해보고 안되면 수동으로 구성할 것 - 체계적으로 일하는 건 좋지만 다른 사람의 일에 병목을 일으켜선 안됨
- [ ] 재욱
- [ ] 찬준

### 다음주 목표
- [ ] 재욱 - 클라우드에 컨테이너 서빙 후 코드 컨버팅
- [ ] 수민 -
- [ ] 찬준 -
- [ ] 나연 -

## 특이사항
언제든 배포가능하고 고도화 가능한 형태로 포트폴리오를 가져가기 위해 프로젝트 종료 후 2주 정도의 시간 투자가 필요
- [x] 프로젝트 종료후 출석에 대해 운영사무실 문의 - 수료후 6개월 가능, 원래 다니던 용산 캠퍼스가 원칙
- [ ] 공공데이터 api 찬준에게 전송
- [ ] 마일스톤 작성 - 미작성시 사유 설명 
- [ ] 어떤 항목을 모니터링 할 것인지 취합 - disk io, 쿼리시간, 스프링 부트 로그

## 기타
* 컨트롤1 2cpu 4G 노드3 2cpu 최소1G / t3.micro는 서울이 아닌 다른 리전에서 프리티어이므로 고려필요, 기본 os가 300m 정도 램 사용
* 멘토링이 늦어질시 일일보고 인원 중간에 빠질것

**Comment by ssedogdoom** [comment]
terraform 코드 다 구성했습니다 . (infra) - 
이 중에서 세부 수정만 하면 되는 상황입니다 
![image](https://github.com/user-attachments/assets/6f7676e0-2abb-470a-838f-3f2fe18db415)

**Comment by ssedogdoom** [comment]
공공데이터 어떤 api를 보내야 하는 지 잘 못들었습니다 
@pcj1792

**Reply by pcj1792** [reply]
자생식물 종별 종자 이미지정보 서비스 입니다

**Comment by ssedogdoom** [comment]
**기타 부분** 
https://jenakim47.tistory.com/102

각 리전 별로 띄울 수 있는 최대 pod가 정해져있습니다.

free tier라 해도 무료로 제공되는 사용량은 한정되어있습니다.
Pod갯수가  최대치를 넘으면 새롭게 Node가 준비가 되어 인스턴스 가격이 더 나가는 상황이 발생할 수 있습니다.
참고 부탁드립니다.
